src/test/java/InternSearch/pages/SearchPageSearch.java

```diff
@@ -0,0 +1,29 @@
+package InternSearch.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+public class SearchPageSearch extends BasePageSearch {
+
+    //Locators
+    @FindBy(css="[type='search']")
+    WebElement searchInput;
```

Comment:
**I recommend:  make all the locators and WebElements private,** because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

```
    @FindBy(css="[type='search']")
    private WebElement searchInput;
```

Reply:
and changed locator to private for restrict access